package.json

```diff
@@ -267,29 +267,30 @@
     "redhat.java"
   ],
   "devDependencies": {
-    "@types/fs-extra": "^5.0.4",
+    "@types/fs-extra": "^5.1.0",
     "@types/glob": "^7.1.1",
-    "@types/lodash": "^4.14.139",
+    "@types/lodash": "^4.14.150",
     "@types/minimatch": "^3.0.3",
-    "@types/mocha": "^5.2.5",
-    "@types/node": "^8.10.36",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^8.10.60",
     "@types/vscode": "1.44.0",
-    "glob": "^7.1.4",
-    "gulp": "^4.0.0",
+    "glob": "^7.1.6",
+    "gulp": "^4.0.2",
     "gulp-copy": "^4.0.1",
-    "gulp-tslint": "^8.1.3",
+    "gulp-tslint": "^8.1.4",
     "mocha": "^7.1.2",
-    "ts-loader": "^5.3.1",
-    "tslint": "^5.11.0",
-    "typescript": "^3.1.6",
-    "vscode-test": "^1.2.0",
-    "webpack": "^4.43.0",
+    "ts-loader": "^5.4.5",
+    "tslint": "^5.20.1",
+    "typescript": "^3.8.3",
+    "vscode-test": "^1.3.0",
+    "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
     "fs-extra": "^7.0.1",
     "lodash": "^4.17.15",
     "minimatch": "^3.0.4",
-    "vscode-extension-telemetry-wrapper": "^0.7.1"
+    "vscode-extension-telemetry-wrapper": "^0.7.1",
+    "vscode-tas-client": "0.0.702"
```

Comment:
You should actually use version `0.0.659` of `vscode-tas-client` for now. `0.0.702` has a bug that I'll resolve later this week.